src/systemd/meson.build

```diff
@@ -58,6 +58,7 @@ if add_languages('cpp', required : false)
         opts += [['c++'],
                  ['c++', '-std=c++98'],
                  ['c++', '-std=c++11']]
+        cxx = meson.get_compiler('cpp')
```

Comment:
I'd prefer to not define `cxx` twice. It's already defined in the main build file. Maybe this should just be
```
if cxx_cmd != ''
         opts += [['c++'],
                 ['c++', '-std=c++98'],
                 ['c++', '-std=c++11']]
         ...
```
?

Reply:
Let me try that

Reply:
Works for me, updated.